server/common/src/main/java/org/eclipse/lsp/cobol/common/utils/PredefinedCopybooks.java

```diff
@@ -51,7 +51,14 @@ public enum Copybook {
     SQLCA {
       @Override
       public String nameForBackend(SQLBackend backend) {
-        return backend == SQLBackend.DATACOM_SERVER ? "SQLCA_DATACOM" : "SQLCA_DB2";
+        switch (backend) {
+          case DB2_SERVER:
+            return "SQLCA_DB2";
+          case DATACOM_SERVER:
+            return "SQLCA_DATACOM";
+          default:
+            return "SQLCA_DB2";
+        }
```

Comment:
This can be reverted.
```suggestion
        return backend == SQLBackend.DATACOM_SERVER ? "SQLCA_DATACOM" : "SQLCA_DB2";
```